### ISSUE TITLE
fix seed in DataLoaders

### ISSUE THREAD

**Issue by wtaisner** [jkarolczak/ligand-classification#59]
according to [docs](https://pytorch.org/docs/stable/notes/randomness.html), make sure that random state in dataloaders in scripts will be fixed to a constant value
